component.yaml

```
@@ -39,6 +39,18 @@ inputs:
   title: Success Transition
   description: "Transition ID or name for the successful step. If the previous state succeeds, you can optionally change the status of your issue."
   helpText:
+- key: "AC_JIRA_REST_API_VERSION"
+  defaultValue: "3"
+  isRequired: true
+  title: Jira REST API Version
+  description: "Specify the REST API version of Jira. Cloud Jira users generally use the latest version (currently `3`) of the API, but Jira enterprise users can use different API versions."
+- key: "AC_JIRA_USE_PAT_AUTH"
+  defaultValue: "false"
+  isRequired: false
+  options: "true,false"
```

Two options were provided but editor type not specified. So in dev env it looks like textfiled. 
```suggestion
  editorType: select
  options: "true,false"
```